src/main/java/utils/api/ApiUtils.java

```diff
@@ -0,0 +1,111 @@
+package utils.api;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import endPoint.Point;
+import lombok.SneakyThrows;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import user.User;
+
+import java.net.URI;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class ApiUtils {
+    @SneakyThrows
+    public static String sendGet(String token, String methodPoint, int statusCod) {
+        HttpGet httpget = new HttpGet(createApiMethodUri(token, methodPoint));
+        CloseableHttpClient httpclient = HttpClients.createDefault();
+        CloseableHttpResponse response = httpclient.execute(httpget);
+        int statusCode = response.getStatusLine().getStatusCode();
+        assertThat(statusCode, equalTo(statusCod));
+        String jsonString;
+        try {
+            jsonString = EntityUtils.toString(response.getEntity());
+        } finally {
+            response.close();
+        }
+        return jsonString;
```

Comment:
Should be moved to the method the works with response. It should be moved to Utils

Reply:
fixed